crates/editor/src/hover_links.rs

```diff
@@ -180,12 +181,30 @@ impl Editor {
                     cx.focus(&self.focus_handle);
                 }
 
-                return self.navigate_to_hover_links(
-                    None,
-                    hovered_link_state.links,
-                    modifiers.alt,
-                    cx,
-                );
+                // exclude links pointing back to the current anchor
+                let current_position = point
+                    .next_valid
```

Comment:
I'm not sure whether `previous_valid` or `next_valid` should be used.

Reply:
Looks like they are the same after trying both.

Reply:
They would only be different if you clicked on an inlay hint or similar. It's probably fine to just use one.